guvnor-ala/guvnor-ala-openshift/guvnor-ala-openshift-ui/guvnor-ala-openshift-ui-api/src/main/resources/org/guvnor/ala/ui/openshift/GuvnorAlaUIOpenShiftAPI.gwt.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!DOCTYPE module PUBLIC "-//Google Inc.//DTD Google Web Toolkit 2.7.0//EN"
+    "http://www.gwtproject.org/doctype/2.7.0/gwt-module.dtd">
+
+<module>
```

Comment:
It's a "place holder" for the upcoming OpenShift provider UI. 
I've added lets say the "complete" standard ui module structure in advance, also to eventually help a future contributor/author of the OpenShift provider. I'd suggest to let it be, and I have this in my notes. When the OpenShift provider is ready and if this project remains empty i'll will remove it. It's under control.